services/agents/postgresql.go

```diff
@@ -47,6 +47,8 @@ func postgresExporterConfig(service *models.Service, exporter *models.Agent, red
 		"--collect.custom_query.mr.directory=/usr/local/percona/pmm2/collectors/custom-queries/postgresql/medium-resolution",
 		"--collect.custom_query.hr.directory=/usr/local/percona/pmm2/collectors/custom-queries/postgresql/high-resolution",
 
+		"--auto-discover-databases",
+		"--exclude-databases=template0,template1,postgres",
```

Comment:
do we really should exclude template0, template1 and postgres?

Reply:
Yes we do. Please check my comment in the ticket.

Also here are some snippets from postgresql documentation: 
_"CREATE DATABASE actually works by copying an existing database. By default, it copies the standard system database named template1. Thus that database is the "template" from which new databases are made. 
...
There is a second standard system database named template0. This database contains the same data as the initial contents of template1, that is, only the standard objects predefined by your version of PostgreSQL."_

Reply:
ok, then. let's fix tests (only the one related to this ticket) and go on

Reply:
Done